src/powershell/README.md

```diff
@@ -17,6 +17,7 @@ Installs PowerShell along with needed dependencies. Useful for base Dockerfiles
 |-----|-----|-----|-----|
 | version | Select or enter a version of PowerShell. | string | latest |
 | modules | Optional comma separated list of PowerShell modules to install. | string | - |
+| powershellProfileURL | Optional URL to download Powershell Profile. | string | - |
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!